OpenEdXMobile/src/main/java/org/edx/mobile/model/api/EnrollmentResponse.kt

```diff
@@ -39,8 +40,12 @@ data class EnrollmentResponse(
                         Gson().fromJson((json as JsonArray), listType)
                     )
                 } else {
+                    val config = (json as JsonObject)
+                        .getAsJsonObject("configs")
+                        .getAsJsonPrimitive("config")
+
                     val appConfig = Gson().fromJson(
-                        (json as JsonObject).get("config"),
+                        JsonParser.parseString(config.asString),
```

Comment:
We can simply use `val appConfig = Gson().fromJson(config.asString, AppConfig::class.java)` here